README.md
Local analysis code for EDM analysis. 
-------------------------------------

Important information:

---> By far, the majority of the work is done in /EDM/CMacros 
---> Base histograms are produced in /gm2/app/users/sgrant/analysis/EDM/ReadNTup, or ReadNTup (github).
---> Compiled code is controlled by Makefile (to run, type "make")
---> Header files /EDM/CMacros/Utils.h and /EDM/CMacros/FancyDraw.h contain some common fitting and plotting code, used by most of the .C files in /EDM/CMacros 
---> Most important fitting code for the EDM is /EDM/CMacros/BlindedEDMFitter.C and /EDM/CMacros/BlindedEDMSimFitter.C
---> The output from the fitters are taken by /EDM/CMacros/GetTiltAngle.C, which applies sorrections are applied calculated the tilt angle. This also prints results in .csv format
---> theta_y vs y acceptance maps are produced by /EDM/CMacros/AcceptanceMaps.C
---> The acceptance correction, used in /EDM/CMacros/GetTiltAngle.C, is calculated in /EDM/CMacros/AcceptanceCorrection.C
---> Toy radial magnetic field measurement self-contained in ToyRadialFieldScan.C ToyRadialFieldScan.h
---> Radial magnetic field measurement fits are performed in /EDM/CMacros/AnalyseBr_nearline.C
---> The rest of the stuff here is mostly just macros to create specific plots 




CMacros/history.txt
    1  curl -O https://raw.githubusercontent.com/MartinSeeler/iterm2-material-design/master/material-design-colors.itermcolors
    2  cd gm2/gain_corrections/inFillGainParam
    3  emacs inFillGainParams_Ep_xtal.cc
    4  emacs
    5  uninstall_oh_my_zsh
    6  cat .bash_profie
    7  cat .bash_profile
    8  cat .bashrc
    9  cat .bash_history
   10  ls -a z*
   11  cat /Users/samuelgrant/.zshrc
   12  sublime /Users/samuelgrant/.zshrc
   13  echo $HOME
   14  echo $HOME/.ssh/config
   15  sublime $HOME/.ssh/config
   16  cp $HOME/.ssh/config shh_config_BK
   17  sublime ssh_config_BK
   18  sublime shh_config_BK
   19  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
   20  ls /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs
   21  ls /Applications/Xcode.app/
   22  xcode-select --install
   23  ls /Library/Developer/CommandLineTools/
   24  ls /Library/Developer/CommandLineTools/SDKs
   25  git -C /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core fetch --unshallow
   26  ssh gm2ucl
   27  git -C /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask fetch --unshallow
   28  ssh-keygen -R g2gateway01
   29  ssh-keygen -R g2gateway01.fnal.gov
   30  ssh -vvv g2gateway01
   31  ssh g2gateway02
   32  ssh g2be3
   33  ssh g2tracker0
   34  brew update root
   35  brew upgrade root
   36  echo $ZSH
   37  ls /Users/samuelgrant/.oh-my-zsh
   38  sh -c "$(curl -fsSL https://raw.github.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"
   39  echo $i
   40  ssh -nNT -L 5002:g2web:5002 G2Muon@g2gateway01.fnal.gov
   41  ssh -nNT -L 5001:g2tracker0-priv:5002 G2Muon@g2gateway01.fnal.gov &
   42  brew link --overwrite --dry-run python@2
   43  jupyter-notebook
   44  ssh -L 5009:g2web:5009 G2Muon@g2gateway01.fnal.gov
   45  ssh -L 5009:g2web:5009 G2Muon@g2gateway01.fnal.gov & 
   46  ssh -L 5009:g2web:5009 G2Muon@g2gateway01.fnal.gov &
   47  sshmc1_old
   48  nano
   49  sublime .bash_profile
   50  sshmc1_new
   51  ssh -nNT -L 8989:gm2-01.dhcp.fnal.gov:80 -L 8081:g2field-be-priv:8080 -L 5002:g2web-priv:5002 -L 3334:gm2-01.dhcp.fnal.gov:3334 -L 7000:g2web:7000 -L 8787:g2be2:8787 -L 5001:g2tracker0-priv:5002 -L 8080:g2be1-priv:8080 -L 4444:gm2-01.dhcp.fnal.gov:4444 -L 4488:gm2-01.dhcp.fnal.gov:4488 -L 4491:gm2-01.dhcp.fnal.gov:4491 -L 4492:gm2-01.dhcp.fnal.gov:4492 -L 3333:g2be2:3333 -L 8781:dbweb5.fnal.gov:8080 -L 8077:192.168.30.7:80 -L 8078:192.168.30.8:80 -L 8079:192.168.30.9:80 -L 2080:g2field-fe1-priv:2080 -L 2081:g2field-fe1-priv:2081 -L 5005:g2field-server-2:5005 -L 8888:g2laser-priv:8080 -L 9002:g2web-priv:9002 -L 9090:g2web-priv:9090 -L 1234:g2web:5005 -L 5009:g2web:5009 -o GSSAPIAuthentication=yes G2Muon@g2gateway01.fnal.gov
   52  ssh -nNT -L 8080:g2be1:8080 G2Muon@g2gateway01.fnal.gov &
   53  ssh -nNT -L 3333:g2be2-priv:3333 G2Muon@g2gateway01.fnal.gov &
   54  ssh -nNT -L 8787:g2be2:8787 G2Muon@g2gateway01.fnal.gov &
   55  ssh -nNT -L 5002:g2web:5002 G2Muon@g2gateway01.fnal.gov & 
   56  ssh -nNT -L 5002:g2web:5002 G2Muon@g2gateway01.fnal.gov &
   57  ssh -nNT -L 5002:g2web:5002 G2Muon@g2gateway01.fnal.gov 
   58  ssh -nNT -L 5009:g2web:5009 G2Muon@g2gateway01.fnal.gov
   59  ssh -nNT -L 1234:g2web:5005 G2Muon@g2gateway01.fnal.gov
   60  ssh -nNT -L 8989:gm2-01.dhcp.fnal.gov:80
   61  ssh -nNT -L 8081:g2field-be-priv:8080 G2Muon@g2gateway01.fnal.gov &
   62  ssh -nNT -L 5009:g2web:5009 G2Muon@g2gateway01.fnal.gov &
   63  killall ssh
   64  echo $ZSH_THEME
   65  sd
   66  lx
   67  cp IOP_sgrant.pptx IOP_sgrant_draft.pptx
   68  cp BlindedEDMSimFitter.C BlindedSimpleEDMSimFitter.C
   69  cp UnblindedEDMSimFitter.C UnblindedSimpleEDMSimFitter.C
   70  open ../Images/MC/dMuSim/5.4e-18/Blinded/fit_mod_wiggle_vertCorr_eQ_eQ.png
   71  open ../Images/MC/dMuSim/5.4e-18/Blinded/S0S12S18_ModuloFit_vertCorr_eQ_eQ_0.png
   72  open ../Images/MC/dMuSim/5.4e-18/Blinded/S0S12S18_ModuloFit_vertCorr_eQ_eQ_1.png
   73  cd ../../Literature
   74  sublime ../Images/Data/RadialFieldEstimation/BeamYPosMonitoring/Run4_Jan/y-pos_calos_meanSub_37970-39100.C
   75  open ../Images/Data/RadialFieldEstimation/BeamYPosMonitoring/Run4_Jan/y-pos_calos_37970-39100.png
   76  open ../Images/Data/RadialFieldEstimation/BeamYPosMonitoring/Run4_Jan/y-pos_37970-39100.png
   77  mv ../Images/Data/RadialFieldEstimation/BeamYPosMonitoring/Run4_Jan/y-pos_37970-39100.C ../Images/Data/RadialFieldEstimation/BeamYPosMonitoring/Run4_Jan/ypos_3797039100.C
   78  sublime  ../Images/Data/RadialFieldEstimation/BeamYPosMonitoring/Run4_Jan/ypos_3797039100.C
   79  root  ../Images/Data/RadialFieldEstimation/BeamYPosMonitoring/Run4_Jan/ypos_3797039100.C
   80  sublime dY_and_dBr.CC
   81  ls -l *C
   82  ./PlotBeamYPosVsRun_Run4_Jan.exe
   83  open ../Images/Data/RadialFieldEstimation/BeamYPosMonitoring/Run4_Jan/y-pos_calos_meanSub_37970-39100.png
   84  mkdir E898_papers
   85  wget https://journals.aps.org/prl/pdf/10.1103/PhysRevLett.126.141801 .
   86  wget https://arxiv.org/pdf/2104.03240.pdf .
   87  wget https://arxiv.org/pdf/2104.03240.pdf
   88  mv PhysRevLett.126.141801 Summary_Run1.pdf
   89  which xcode
   90  cd photography/Yosemite
   91  open IMG_4024.JPG
   92  cd RadialFieldEstimation/BeamYPosMonitoring/Input/
   93  ls Run2B
   94  ls Run2C
   95  ls Run2
   96  ls Run1
   97  cd CaloRunNumbers
   98  cat run1.txt | sort -V | tee run1.txt
   99  cat gm2pro_daq_full_run1_60h_5039A_GLdocDB16021-v2.txt | wc -l
  100  cat gm2pro_daq_full_run1_*.txt | wc -l
  101  cat run2.txt | sort -V | tee run2.txt

tmp/4.C
void 4()
{
//=========Macro generated from canvas: c/c
//=========  (Mon Sep  6 10:33:32 2021) by ROOT version 6.22/08
   TCanvas *c = new TCanvas("c", "c",0,0,800,600);
   c->SetHighLightColor(2);
   c->Range(0,0,1,1);
   c->SetFillColor(0);
   c->SetBorderMode(0);
   c->SetBorderSize(2);
   c->SetFrameBorderMode(0);
   
   TH1D *prY__4 = new TH1D("prY__4","4",500,-25,25);
   prY__4->SetBinContent(0,1811);
   prY__4->SetBinContent(1,28);
   prY__4->SetBinContent(2,28);
   prY__4->SetBinContent(3,18);
   prY__4->SetBinContent(4,25);
   prY__4->SetBinContent(5,21);
   prY__4->SetBinContent(6,23);
   prY__4->SetBinContent(7,25);
   prY__4->SetBinContent(8,28);
   prY__4->SetBinContent(9,35);
   prY__4->SetBinContent(10,21);
   prY__4->SetBinContent(11,33);
   prY__4->SetBinContent(12,30);
   prY__4->SetBinContent(13,34);
   prY__4->SetBinContent(14,27);
   prY__4->SetBinContent(15,23);
   prY__4->SetBinContent(16,38);
   prY__4->SetBinContent(17,20);
   prY__4->SetBinContent(18,27);
   prY__4->SetBinContent(19,37);
   prY__4->SetBinContent(20,40);
   prY__4->SetBinContent(21,35);
   prY__4->SetBinContent(22,40);
   prY__4->SetBinContent(23,26);
   prY__4->SetBinContent(24,27);
   prY__4->SetBinContent(25,40);
   prY__4->SetBinContent(26,34);
   prY__4->SetBinContent(27,46);
   prY__4->SetBinContent(28,30);
   prY__4->SetBinContent(29,46);
   prY__4->SetBinContent(30,32);
   prY__4->SetBinContent(31,32);
   prY__4->SetBinContent(32,43);
   prY__4->SetBinContent(33,35);
   prY__4->SetBinContent(34,34);
   prY__4->SetBinContent(35,52);
   prY__4->SetBinContent(36,43);
   prY__4->SetBinContent(37,37);
   prY__4->SetBinContent(38,34);
   prY__4->SetBinContent(39,44);
   prY__4->SetBinContent(40,29);
   prY__4->SetBinContent(41,42);
   prY__4->SetBinContent(42,46);
   prY__4->SetBinContent(43,32);
   prY__4->SetBinContent(44,51);
   prY__4->SetBinContent(45,42);
   prY__4->SetBinContent(46,59);
   prY__4->SetBinContent(47,63);
   prY__4->SetBinContent(48,56);
   prY__4->SetBinContent(49,53);
   prY__4->SetBinContent(50,47);
   prY__4->SetBinContent(51,53);
   prY__4->SetBinContent(52,44);
   prY__4->SetBinContent(53,62);
   prY__4->SetBinContent(54,63);
   prY__4->SetBinContent(55,73);
   prY__4->SetBinContent(56,65);
   prY__4->SetBinContent(57,60);
   prY__4->SetBinContent(58,71);
   prY__4->SetBinContent(59,54);
   prY__4->SetBinContent(60,67);
   prY__4->SetBinContent(61,60);
   prY__4->SetBinContent(62,56);
   prY__4->SetBinContent(63,67);
   prY__4->SetBinContent(64,74);
   prY__4->SetBinContent(65,63);
   prY__4->SetBinContent(66,62);
   prY__4->SetBinContent(67,78);
   prY__4->SetBinContent(68,60);
   prY__4->SetBinContent(69,76);
   prY__4->SetBinContent(70,67);
   prY__4->SetBinContent(71,75);
   prY__4->SetBinContent(72,83);
   prY__4->SetBinContent(73,110);
   prY__4->SetBinContent(74,86);
   prY__4->SetBinContent(75,101);
   prY__4->SetBinContent(76,101);
   prY__4->SetBinContent(77,82);
   prY__4->SetBinContent(78,97);
   prY__4->SetBinContent(79,92);
   prY__4->SetBinContent(80,91);
   prY__4->SetBinContent(81,124);
   prY__4->SetBinContent(82,99);
   prY__4->SetBinContent(83,100);
   prY__4->SetBinContent(84,94);
   prY__4->SetBinContent(85,121);
   prY__4->SetBinContent(86,110);
   prY__4->SetBinContent(87,131);
   prY__4->SetBinContent(88,105);
   prY__4->SetBinContent(89,101);
   prY__4->SetBinContent(90,102);
   prY__4->SetBinContent(91,114);
   prY__4->SetBinContent(92,138);
   prY__4->SetBinContent(93,94);
   prY__4->SetBinContent(94,134);
   prY__4->SetBinContent(95,122);
   prY__4->SetBinContent(96,146);
   prY__4->SetBinContent(97,130);
   prY__4->SetBinContent(98,117);
   prY__4->SetBinContent(99,148);
   prY__4->SetBinContent(100,152);
   prY__4->SetBinContent(101,147);
   prY__4->SetBinContent(102,137);
   prY__4->SetBinContent(103,169);
   prY__4->SetBinContent(104,147);
   prY__4->SetBinContent(105,174);
   prY__4->SetBinContent(106,154);
   prY__4->SetBinContent(107,165);
   prY__4->SetBinContent(108,170);
   prY__4->SetBinContent(109,186);
   prY__4->SetBinContent(110,185);
   prY__4->SetBinContent(111,211);
   prY__4->SetBinContent(112,180);
   prY__4->SetBinContent(113,212);
   prY__4->SetBinContent(114,182);
   prY__4->SetBinContent(115,204);
   prY__4->SetBinContent(116,211);
   prY__4->SetBinContent(117,219);
   prY__4->SetBinContent(118,236);
   prY__4->SetBinContent(119,232);
   prY__4->SetBinContent(120,250);
   prY__4->SetBinContent(121,257);
   prY__4->SetBinContent(122,252);
   prY__4->SetBinContent(123,287);
   prY__4->SetBinContent(124,282);
   prY__4->SetBinContent(125,283);
   prY__4->SetBinContent(126,263);
   prY__4->SetBinContent(127,310);
   prY__4->SetBinContent(128,282);
   prY__4->SetBinContent(129,298);
   prY__4->SetBinContent(130,314);
   prY__4->SetBinContent(131,319);
   prY__4->SetBinContent(132,322);
   prY__4->SetBinContent(133,357);
   prY__4->SetBinContent(134,360);
   prY__4->SetBinContent(135,380);
   prY__4->SetBinContent(136,371);
   prY__4->SetBinContent(137,403);
   prY__4->SetBinContent(138,420);
   prY__4->SetBinContent(139,458);
   prY__4->SetBinContent(140,502);
   prY__4->SetBinContent(141,444);
   prY__4->SetBinContent(142,480);
   prY__4->SetBinContent(143,458);
   prY__4->SetBinContent(144,476);
   prY__4->SetBinContent(145,471);
   prY__4->SetBinContent(146,542);
   prY__4->SetBinContent(147,558);
   prY__4->SetBinContent(148,548);
   prY__4->SetBinContent(149,598);
   prY__4->SetBinContent(150,578);
   prY__4->SetBinContent(151,573);
   prY__4->SetBinContent(152,645);
   prY__4->SetBinContent(153,643);
   prY__4->SetBinContent(154,701);
   prY__4->SetBinContent(155,700);
   prY__4->SetBinContent(156,714);
   prY__4->SetBinContent(157,750);
   prY__4->SetBinContent(158,792);
   prY__4->SetBinContent(159,840);
   prY__4->SetBinContent(160,855);
   prY__4->SetBinContent(161,910);
   prY__4->SetBinContent(162,883);
   prY__4->SetBinContent(163,977);
   prY__4->SetBinContent(164,949);
   prY__4->SetBinContent(165,1015);
   prY__4->SetBinContent(166,1018);
   prY__4->SetBinContent(167,1108);
   prY__4->SetBinContent(168,1196);
   prY__4->SetBinContent(169,1231);
   prY__4->SetBinContent(170,1226);
   prY__4->SetBinContent(171,1262);
   prY__4->SetBinContent(172,1302);
   prY__4->SetBinContent(173,1361);
   prY__4->SetBinContent(174,1345);
   prY__4->SetBinContent(175,1502);
   prY__4->SetBinContent(176,1560);
   prY__4->SetBinContent(177,1650);
   prY__4->SetBinContent(178,1679);
   prY__4->SetBinContent(179,1762);
   prY__4->SetBinContent(180,1839);
   prY__4->SetBinContent(181,1937);
   prY__4->SetBinContent(182,1907);
   prY__4->SetBinContent(183,2099);
   prY__4->SetBinContent(184,2228);
   prY__4->SetBinContent(185,2297);
   prY__4->SetBinContent(186,2391);
   prY__4->SetBinContent(187,2485);
   prY__4->SetBinContent(188,2612);
   prY__4->SetBinContent(189,2779);
   prY__4->SetBinContent(190,2945);
   prY__4->SetBinContent(191,3004);
   prY__4->SetBinContent(192,3305);
   prY__4->SetBinContent(193,3400);
   prY__4->SetBinContent(194,3451);
   prY__4->SetBinContent(195,3741);
   prY__4->SetBinContent(196,3750);
   prY__4->SetBinContent(197,4078);
   prY__4->SetBinContent(198,4278);
   prY__4->SetBinContent(199,4421);
   prY__4->SetBinContent(200,4680);
   prY__4->SetBinContent(201,4989);
   prY__4->SetBinContent(202,5310);
   prY__4->SetBinContent(203,5465);
   prY__4->SetBinContent(204,5746);
   prY__4->SetBinContent(205,6062);
   prY__4->SetBinContent(206,6345);
   prY__4->SetBinContent(207,6847);
   prY__4->SetBinContent(208,7121);
   prY__4->SetBinContent(209,7566);
   prY__4->SetBinContent(210,8050);
   prY__4->SetBinContent(211,8372);
   prY__4->SetBinContent(212,9016);
   prY__4->SetBinContent(213,9455);
   prY__4->SetBinContent(214,9918);
   prY__4->SetBinContent(215,10894);
   prY__4->SetBinContent(216,11156);
   prY__4->SetBinContent(217,11926);
   prY__4->SetBinContent(218,12581);
   prY__4->SetBinContent(219,13380);
   prY__4->SetBinContent(220,14543);
   prY__4->SetBinContent(221,15156);
   prY__4->SetBinContent(222,16281);
   prY__4->SetBinContent(223,17424);
   prY__4->SetBinContent(224,18518);
   prY__4->SetBinContent(225,20026);
   prY__4->SetBinContent(226,21088);
   prY__4->SetBinContent(227,22972);
   prY__4->SetBinContent(228,24740);
   prY__4->SetBinContent(229,26580);
   prY__4->SetBinContent(230,28837);
   prY__4->SetBinContent(231,31324);
   prY__4->SetBinContent(232,34112);
   prY__4->SetBinContent(233,37107);
   prY__4->SetBinContent(234,39821);
   prY__4->SetBinContent(235,44224);
   prY__4->SetBinContent(236,47897);
   prY__4->SetBinContent(237,52357);
   prY__4->SetBinContent(238,57636);
   prY__4->SetBinContent(239,62563);
   prY__4->SetBinContent(240,69492);
   prY__4->SetBinContent(241,75107);
   prY__4->SetBinContent(242,82532);
   prY__4->SetBinContent(243,88774);
   prY__4->SetBinContent(244,95316);
   prY__4->SetBinContent(245,102269);
   prY__4->SetBinContent(246,108338);
   prY__4->SetBinContent(247,114555);
   prY__4->SetBinContent(248,118086);
   prY__4->SetBinContent(249,121728);
   prY__4->SetBinContent(250,123443);
   prY__4->SetBinContent(251,123383);
   prY__4->SetBinContent(252,121996);
   prY__4->SetBinContent(253,118895);
   prY__4->SetBinContent(254,113430);
   prY__4->SetBinContent(255,108049);
   prY__4->SetBinContent(256,102110);
   prY__4->SetBinContent(257,95590);
   prY__4->SetBinContent(258,88748);
   prY__4->SetBinContent(259,81623);
   prY__4->SetBinContent(260,74628);
   prY__4->SetBinContent(261,68597);
   prY__4->SetBinContent(262,62759);
   prY__4->SetBinContent(263,57414);
   prY__4->SetBinContent(264,52451);
   prY__4->SetBinContent(265,47583);
   prY__4->SetBinContent(266,44017);
   prY__4->SetBinContent(267,39854);
   prY__4->SetBinContent(268,36774);
   prY__4->SetBinContent(269,33893);
   prY__4->SetBinContent(270,31161);
   prY__4->SetBinContent(271,28473);
   prY__4->SetBinContent(272,26626);
   prY__4->SetBinContent(273,24686);
   prY__4->SetBinContent(274,22752);
   prY__4->SetBinContent(275,21151);
   prY__4->SetBinContent(276,19762);
   prY__4->SetBinContent(277,18448);
   prY__4->SetBinContent(278,17237);
   prY__4->SetBinContent(279,16099);
   prY__4->SetBinContent(280,15123);
   prY__4->SetBinContent(281,14293);
   prY__4->SetBinContent(282,13295);
   prY__4->SetBinContent(283,12571);
   prY__4->SetBinContent(284,11728);
   prY__4->SetBinContent(285,11225);
   prY__4->SetBinContent(286,10437);
   prY__4->SetBinContent(287,9849);
   prY__4->SetBinContent(288,9307);
   prY__4->SetBinContent(289,9007);
   prY__4->SetBinContent(290,8411);
   prY__4->SetBinContent(291,7745);
   prY__4->SetBinContent(292,7367);
   prY__4->SetBinContent(293,7142);
   prY__4->SetBinContent(294,6569);
   prY__4->SetBinContent(295,6368);
   prY__4->SetBinContent(296,6097);
   prY__4->SetBinContent(297,5783);
   prY__4->SetBinContent(298,5376);
   prY__4->SetBinContent(299,5215);
   prY__4->SetBinContent(300,4794);
   prY__4->SetBinContent(301,4825);
   prY__4->SetBinContent(302,4491);
   prY__4->SetBinContent(303,4219);
   prY__4->SetBinContent(304,3926);
   prY__4->SetBinContent(305,3771);
   prY__4->SetBinContent(306,3571);
   prY__4->SetBinContent(307,3328);
   prY__4->SetBinContent(308,3213);
   prY__4->SetBinContent(309,3090);
   prY__4->SetBinContent(310,3016);
   prY__4->SetBinContent(311,2826);
   prY__4->SetBinContent(312,2820);
   prY__4->SetBinContent(313,2560);
   prY__4->SetBinContent(314,2400);
   prY__4->SetBinContent(315,2344);
   prY__4->SetBinContent(316,2239);
   prY__4->SetBinContent(317,2144);
   prY__4->SetBinContent(318,1983);
   prY__4->SetBinContent(319,1987);
   prY__4->SetBinContent(320,1831);
   prY__4->SetBinContent(321,1843);
   prY__4->SetBinContent(322,1715);
   prY__4->SetBinContent(323,1660);
   prY__4->SetBinContent(324,1573);
   prY__4->SetBinContent(325,1535);
   prY__4->SetBinContent(326,1467);
   prY__4->SetBinContent(327,1402);
   prY__4->SetBinContent(328,1323);
   prY__4->SetBinContent(329,1265);
   prY__4->SetBinContent(330,1199);
   prY__4->SetBinContent(331,1196);
   prY__4->SetBinContent(332,1135);
   prY__4->SetBinContent(333,1079);
   prY__4->SetBinContent(334,1013);
   prY__4->SetBinContent(335,948);
   prY__4->SetBinContent(336,985);
   prY__4->SetBinContent(337,939);
   prY__4->SetBinContent(338,887);
   prY__4->SetBinContent(339,853);
   prY__4->SetBinContent(340,826);
   prY__4->SetBinContent(341,800);
   prY__4->SetBinContent(342,803);
   prY__4->SetBinContent(343,740);
   prY__4->SetBinContent(344,721);
   prY__4->SetBinContent(345,710);
   prY__4->SetBinContent(346,702);
   prY__4->SetBinContent(347,627);
   prY__4->SetBinContent(348,583);
   prY__4->SetBinContent(349,609);
   prY__4->SetBinContent(350,577);
   prY__4->SetBinContent(351,589);
   prY__4->SetBinContent(352,533);
   prY__4->SetBinContent(353,559);
   prY__4->SetBinContent(354,506);
   prY__4->SetBinContent(355,464);
   prY__4->SetBinContent(356,484);
   prY__4->SetBinContent(357,427);
   prY__4->SetBinContent(358,431);
   prY__4->SetBinContent(359,405);
   prY__4->SetBinContent(360,460);
   prY__4->SetBinContent(361,400);
   prY__4->SetBinContent(362,393);
   prY__4->SetBinContent(363,340);
   prY__4->SetBinContent(364,403);
   prY__4->SetBinContent(365,349);
   prY__4->SetBinContent(366,313);
   prY__4->SetBinContent(367,317);
   prY__4->SetBinContent(368,328);
   prY__4->SetBinContent(369,333);
   prY__4->SetBinContent(370,311);
   prY__4->SetBinContent(371,316);
   prY__4->SetBinContent(372,302);
   prY__4->SetBinContent(373,275);
   prY__4->SetBinContent(374,269);
   prY__4->SetBinContent(375,305);
   prY__4->SetBinContent(376,278);
   prY__4->SetBinContent(377,271);
   prY__4->SetBinContent(378,235);
   prY__4->SetBinContent(379,199);
   prY__4->SetBinContent(380,245);
   prY__4->SetBinContent(381,206);
   prY__4->SetBinContent(382,223);
   prY__4->SetBinContent(383,202);
   prY__4->SetBinContent(384,191);
   prY__4->SetBinContent(385,189);
   prY__4->SetBinContent(386,203);
   prY__4->SetBinContent(387,183);
   prY__4->SetBinContent(388,193);
   prY__4->SetBinContent(389,190);
   prY__4->SetBinContent(390,196);
   prY__4->SetBinContent(391,167);
   prY__4->SetBinContent(392,168);
   prY__4->SetBinContent(393,190);
   prY__4->SetBinContent(394,186);
   prY__4->SetBinContent(395,149);
   prY__4->SetBinContent(396,146);
   prY__4->SetBinContent(397,138);
   prY__4->SetBinContent(398,142);
   prY__4->SetBinContent(399,135);
   prY__4->SetBinContent(400,149);
   prY__4->SetBinContent(401,134);
   prY__4->SetBinContent(402,121);
   prY__4->SetBinContent(403,111);
   prY__4->SetBinContent(404,126);
   prY__4->SetBinContent(405,118);
   prY__4->SetBinContent(406,104);
   prY__4->SetBinContent(407,131);
   prY__4->SetBinContent(408,117);
   prY__4->SetBinContent(409,106);
   prY__4->SetBinContent(410,100);
   prY__4->SetBinContent(411,100);
   prY__4->SetBinContent(412,112);
   prY__4->SetBinContent(413,113);
   prY__4->SetBinContent(414,96);
   prY__4->SetBinContent(415,90);
   prY__4->SetBinContent(416,97);
   prY__4->SetBinContent(417,103);
   prY__4->SetBinContent(418,103);
   prY__4->SetBinContent(419,83);
   prY__4->SetBinContent(420,85);
   prY__4->SetBinContent(421,78);
   prY__4->SetBinContent(422,90);
   prY__4->SetBinContent(423,73);
   prY__4->SetBinContent(424,76);
   prY__4->SetBinContent(425,73);
   prY__4->SetBinContent(426,83);
   prY__4->SetBinContent(427,57);
   prY__4->SetBinContent(428,60);
   prY__4->SetBinContent(429,81);
   prY__4->SetBinContent(430,62);
   prY__4->SetBinContent(431,79);
   prY__4->SetBinContent(432,53);
   prY__4->SetBinContent(433,63);
   prY__4->SetBinContent(434,57);
   prY__4->SetBinContent(435,53);
   prY__4->SetBinContent(436,61);
   prY__4->SetBinContent(437,59);
   prY__4->SetBinContent(438,56);
   prY__4->SetBinContent(439,57);
   prY__4->SetBinContent(440,44);
   prY__4->SetBinContent(441,60);
   prY__4->SetBinContent(442,65);
   prY__4->SetBinContent(443,46);
   prY__4->SetBinContent(444,40);
   prY__4->SetBinContent(445,50);
   prY__4->SetBinContent(446,47);
   prY__4->SetBinContent(447,53);
   prY__4->SetBinContent(448,42);
   prY__4->SetBinContent(449,49);
   prY__4->SetBinContent(450,37);
   prY__4->SetBinContent(451,38);
   prY__4->SetBinContent(452,52);
   prY__4->SetBinContent(453,43);
   prY__4->SetBinContent(454,41);
   prY__4->SetBinContent(455,37);
   prY__4->SetBinContent(456,39);
   prY__4->SetBinContent(457,31);
   prY__4->SetBinContent(458,39);
   prY__4->SetBinContent(459,42);
   prY__4->SetBinContent(460,31);
   prY__4->SetBinContent(461,37);
   prY__4->SetBinContent(462,34);
   prY__4->SetBinContent(463,35);
   prY__4->SetBinContent(464,38);
   prY__4->SetBinContent(465,34);
   prY__4->SetBinContent(466,29);
   prY__4->SetBinContent(467,30);
   prY__4->SetBinContent(468,29);
   prY__4->SetBinContent(469,31);
   prY__4->SetBinContent(470,33);
   prY__4->SetBinContent(471,39);
   prY__4->SetBinContent(472,29);
   prY__4->SetBinContent(473,32);
   prY__4->SetBinContent(474,29);
   prY__4->SetBinContent(475,35);
   prY__4->SetBinContent(476,37);
   prY__4->SetBinContent(477,22);
   prY__4->SetBinContent(478,21);
   prY__4->SetBinContent(479,39);
   prY__4->SetBinContent(480,20);
   prY__4->SetBinContent(481,22);
   prY__4->SetBinContent(482,16);
   prY__4->SetBinContent(483,33);
   prY__4->SetBinContent(484,23);
   prY__4->SetBinContent(485,23);
   prY__4->SetBinContent(486,20);
   prY__4->SetBinContent(487,18);
   prY__4->SetBinContent(488,19);
   prY__4->SetBinContent(489,8);
   prY__4->SetBinContent(490,13);
   prY__4->SetBinContent(491,21);
   prY__4->SetBinContent(492,26);
   prY__4->SetBinContent(493,18);
   prY__4->SetBinContent(494,16);
   prY__4->SetBinContent(495,27);
   prY__4->SetBinContent(496,15);
   prY__4->SetBinContent(497,13);
   prY__4->SetBinContent(498,18);
   prY__4->SetBinContent(499,16);
   prY__4->SetBinContent(500,14);
   prY__4->SetBinContent(501,1348);
   prY__4->SetEntries(4011125);
   
   TPaveStats *ptstats = new TPaveStats(0.78,0.735,0.98,0.935,"brNDC");
   ptstats->SetName("stats");
   ptstats->SetBorderSize(1);
   ptstats->SetFillColor(0);
   ptstats->SetTextAlign(12);
   ptstats->SetTextFont(42);
   TText *ptstats_LaTex = ptstats->AddText("Entries = 5316378");
   ptstats_LaTex = ptstats->AddText("Mean  = -0.005486 #pm 0.001224");
   ptstats_LaTex = ptstats->AddText("Std Dev   =  2.822 #pm 0.0008658");
   ptstats->SetOptStat(2210);
   ptstats->SetOptFit(0);
   ptstats->Draw();
   prY__4->GetListOfFunctions()->Add(ptstats);
   ptstats->SetParent(prY__4);
   prY__4->GetXaxis()->SetTitle("True #minus reco #theta_{y} [mrad]");
   prY__4->GetXaxis()->SetRange(1,500);
   prY__4->GetXaxis()->CenterTitle(true);
   prY__4->GetXaxis()->SetLabelFont(42);
   prY__4->GetXaxis()->SetTitleSize(0.04);
   prY__4->GetXaxis()->SetTitleOffset(1.1);
   prY__4->GetXaxis()->SetTitleFont(42);
   prY__4->GetYaxis()->CenterTitle(true);
   prY__4->GetYaxis()->SetNdivisions(4000510);
   prY__4->GetYaxis()->SetLabelFont(42);
   prY__4->GetYaxis()->SetTitleSize(0.04);
   prY__4->GetYaxis()->SetTitleOffset(1.1);
   prY__4->GetYaxis()->SetTitleFont(42);
   prY__4->GetZaxis()->SetLabelFont(42);
   prY__4->GetZaxis()->SetTitleOffset(1);
   prY__4->GetZaxis()->SetTitleFont(42);
   prY__4->Draw("HIST");
   c->Modified();
   c->cd();
   c->SetSelected(c);
}
